api/client.go

```diff
@@ -1466,3 +1577,14 @@ func (w *replicationStateStore) states() []string {
 	copy(c, w.store)
 	return c
 }
+
+// tokenCheck will check for non-printable characters to prevent a call that will fail at the api
+func tokenCheck(t string) error {
```

Comment:
Would `checkToken` or `validateToken` be a better name?

Reply:
Yes. They both would.